.github/workflows/CypressImage.yaml

```diff
@@ -0,0 +1,53 @@
+name: Cypress Build & Push
+env:
+  DATALAND_SKYMINDERCLIENT_TOKEN: ${{ secrets.DATALAND_SKYMINDERCLIENT_TOKEN }}
+  DATALAND_SKYMINDERCLIENT_USER: ${{ secrets.DATALAND_SKYMINDERCLIENT_USER }}
+  DATALAND_EDC_TOKEN: ${{ secrets.DATALAND_EDC_TOKEN }}
+  DATALAND_EDC_USER: ${{ secrets.DATALAND_EDC_USER }}
+  SKYMINDER_URL: ${{ secrets.SKYMINDER_URL }}
+  SKYMINDER_PW: ${{ secrets.SKYMINDER_PW }}
+  SKYMINDER_USER: ${{ secrets.SKYMINDER_USER }}
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - 'main'
```

Comment:
```
  push:
    branches:
      - '**'
  pull_request:
    branches:
      - 'main'
```
kann raus denke ich

Reply:
entfernt

Reply:
Ich habe es auch in der `TemurinImage.yaml` rausgelöscht.
Passt oder? Weil dort starten wir das Ding ja auch nur manuell mit dem Workflow Dispatch.

Reply:
ja klar

Reply:
ok